packages/custom-field/test/custom-field.test.js

```diff
@@ -64,8 +64,8 @@ describe('custom field', () => {
       labelElement = customField.querySelector('[slot="label"]');
     });
 
-    it('should be empty by default', () => {
-      expect(customField.label).to.equal('');
+    it('should be undefined by default', () => {
+      expect(customField.label).to.be.undefined;
```

Comment:
The label property is no longer filled with `_labelNode.textContent` at initialization.